masterlist.yaml

```
@@ -686,6 +686,38 @@ globals:
     subs: [ '**Fores New Idles in Skyrim** or **Nemesis Unlimited Behavior Engine**' ]
     condition: 'file("Nemesis_Engine/Nemesis Unlimited Behavior Engine.exe") and file("tools/GenerateFNIS_for_Users/GenerateFNISforUsers.exe") and not file("scripts/FNISCreatureVersion.pex")'
 
+  # Mod Series Patch Collections
+  - type: say
+    content: 'It is recommended that you check the following compatibility patch compilation for %1%: %2%'
+    subs: 
+      - 'JK''s Interiors'
+      - '[JK''s Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/35910/)'
+    condition: 'active("JK''?s (Angelines Aromatics|Arcadia''s Cauldron|Arnleif and Sons Trading Company|Bee and Barb|Belethor''s General Goods|Bits and Pieces|Candlehearth Hall|Elgrims Elixirs|Haelga''s Bunkhouse|New Gnisis Cornerclub|Radiant Raiment|Riverwood Trader|Sadris Used Wares|Septimus Signus''s Outpost|Silver-Blood Inn|Sinderion''s Field Laboratory|Sleeping Giant Inn|Temple of Dibella|Temple of Kynareth|Temple of Mara|Temple of Talos|Temple of the Divines|The Bannered Mare|The Drunken Huntsman|The Hag''s Cure|The Pawned Prawn|The Ragged Flagon|The Winking Skeever|Warmaiden''s|White Phial)\.esp") or active("JK''?s (Blue Palace|Dragonsreach|Mistveil Keep|Palace of the Kings|Understone Keep)\.esp") and not active("Palaces Castles Enhanced.esp")'
+  - type: say
+    content: 'It is recommended that you check the following compatibility patch compilation for %1%: %2%'
+    subs: 
+      - 'JK''s Guild HQ Interiors'
+      - '[JK''s Guild HQ Interiors Patch Collection](https://www.nexusmods.com/skyrimspecialedition/mods/61416/)'
+    condition: 'active("JK''?s (Bards CollegeHigh Hrothgar|Jorrvaskr|Sky Haven Temple)\.esp") or active("JK''s Castle Dour.esp") and not active("JKs Castle Dour - PCE Patch.esp") or active("JK''?s College of Winterhold.esp") and not active("JKs College of Winterhold - ICOW Patch.esp") and not active("JKs College of Winterhold - OCW Patch.esp")'
```

This condition string has a couple of small issues. Also, it'd be better to check for PCE, ICoW, & OCW instead of their patches as presumably you'd want users to go to the combo patch collections when they have those mods active instead:

```suggestion
    condition: 'active("JK''?s (The Bards College|High Hrothgar|Jorrvaskr|Sky Haven Temple)\.esp") or active("JK''s Castle Dour.esp") and not active("Palaces Castles Enhanced.esp") or active("JK''s College of Winterhold.esp") and not active("CollegeOfWinterholdImmersive.esp") and not active("OCW_Obscure''s_CollegeofWinterhold.esp")'
```